samples/jenkins/README.md

```diff
@@ -1,5 +1,370 @@
 # Jenkins Integration
+The sample demonstrates `cpdctl` integration with Jenkins CI/CD system, by the implementation of the 
+AI Lifecycle Management scenario described in the [article](https://medium.com/@rafal.bigaj/12713ef04dc5). 
 
-Sample Jenkins pipelines demonstrating `cpdctl` use in the external CI/CD system.
+## Prerequisites
+1. [IBM Cloud Pak for Data](https://www.ibm.com/products/cloud-pak-for-data), version 4.0.2 or above. 
+2. [CPDCTL](https://github.com/IBM/cpdctl), version 1.1.15 or above.
+3. Jenkins server with [Pipeline](https://plugins.jenkins.io/workflow-aggregator/), [GitHub](https://plugins.jenkins.io/github/) and [Copy Artifact](https://plugins.jenkins.io/copyartifact/) plugins installed. It is also possible to use Jenkins official
+[Docker image](https://hub.docker.com/_/jenkins) to [setup](https://www.jenkins.io/doc/book/installing/docker/) an isolated
+Jenkins instance.
 
+### CPDCTL configuration
+Detailed installation procedure of CPDCTL can be found in the project [README](https://github.com/IBM/cpdctl/#installation). 
+Configuration of the environment requires url of the CPD cluster, username and password, or username and apikey when
+IAM integration is [enabled](https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=tasks-integrating-iam-service).
+To setup the cluster as a target environment in the CPDCTL cli please follow the steps:
```

Comment:
Would replace `CPDCTL` (as per earlier comment) with `cpdctl` and `cli` with `CLI`.
Use of word `please` is discouraged per IBM documentation guidelines.